pages/api/discord-thread.ts

```diff
@@ -136,8 +147,11 @@ async function fetchDiscordThread(
   l1Request: L1Request
 ): Promise<VoteDiscussionT> {
   // First, fetch all messages in the evidence rational channel.
-  const threadMsg = await getDiscordMessages(evidenceRationalDiscordChannelId);
-
+  // we dont really need to go back far in history
+  const threadMsg = await getDiscordMessages(
```

Comment:
this is the main change. we dont paginate old threads, since there are so many, this reduces the number of fetches with pagination.  we dont need to go back more than 50 threads back

Reply:
Does this mean that if you go back to 50 votes back, the discord threads will no longer show up?

If there are 51 votes, will this also break?

Reply:
thats right, it will only go back so far